cursorless-talon/src/modifiers/modifiers.py

```diff
@@ -14,6 +14,9 @@
     "just": "toRawSelection",
     "leading": "leading",
     "trailing": "trailing",
+    "content": "content",
+    "empty": "empty",
+    "whitespace": "whitespace",
```

Comment:
Now that I see the impl for these two, I'm kind of struggling to see the use case.  I wonder if we should have them disabled by default, or just remove them for now?  Not sure it's worth polluting our spoken forms with things that we may never use, even if they are trivial to implement 🤔

I think `"content"` is great.  The term is a bit unintuitive, because just the word `"content"` on its own doesn't imply to me that it is filtering.

I wonder if default spoken form should be a bit longer, given it's unlikely to be a super common use case.  Eg `"only content"` or `"keep content"`

We could by default prefix all of them with `"keep"` or `"only"`, then it's not such a big deal that we're polluting the spoken forms as it's a bit longer